klarna-on-demand/src/main/java/com/klarna/ondemand/crypto/KeyStoreCryptoImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.klarna.ondemand.crypto;
+
+
+import android.annotation.TargetApi;
+import android.os.Build;
+import android.security.KeyPairGeneratorSpec;
+import android.util.Base64;
+
+import java.io.IOException;
+import java.math.BigInteger;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.NoSuchProviderException;
+import java.security.UnrecoverableEntryException;
+import java.security.cert.CertificateException;
+import java.util.Calendar;
+import java.util.Date;
+
+import javax.security.auth.x500.X500Principal;
+
+class KeyStoreCryptoImpl extends CryptoBase {
+
+    private static final String ANDROID_KEY_STORE = "AndroidKeyStore";
```

Comment:
Just wondering, could we have named this however we wanted? If the answer is yes, why did we choose this name? If not, are you sure this constant isn't already defined somewhere else?